code/game/objects/items/devices/instruments.dm

```diff
@@ -143,7 +143,7 @@
 	instrumentId = "saxophone"
 	force = 0
 	attack_verb = list("played","jazzed","saxxed","mourned","dooted")
-
+	too_spooky = TRUE

```

Comment:
Why did you add this here if you didn't remove the flag in the base /instrument item

Reply:
To allow it to spawn new instruments.